{"run_number": "1747761235085033", "step_name": "ingenieria_de_datos", "task_id": "2", "attempt_id": 0, "name": "_exception", "content_type": "gzip+pickle-v4", "type": "metaflow.artifact", "sha": "dbc1aaddc8b7343d6d33b34edcf608b8f8801918", "ds_type": "local", "location": ":root:/home/clau/proyectos/metaflow/Practica-SIE-METAFLOW/.metaflow", "flow_id": "InitFlow", "user_name": "clau", "tags": [], "system_tags": ["user:clau", "runtime:dev", "metaflow_version:2.15.6", "python_version:3.12.3"], "ts_epoch": 1747761237447}